app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const request = require('request-promise');
+const errors = require('errors.js');
+const logger = require('/logger/index.js');
```

Comment:
It's not necessary import index, `require('/logger')` requires directly `/logger/index.js`